samples/net/google_iot_device/src/config.c.in

```diff
@@ -0,0 +1,20 @@
+#include "iotc.h"
+
+/* CHANGE THIS */
+const char* iotc_project_id = "macro-precinct-211108";
```

Comment:
pls use template strings, something like:
`const char* iotc_project_id = YOUR_GOOGLE_CLOUD_IOT_PROJECT_ID`
Additionally would be good to leave example lines for users. Like this one but with a fictional project name / content. Actually for project name it isn't that useful, but for the `iotc_device_path` I am sure it'll come handy for the first time visitor.

Reply:
> pls use template strings, something like:

Ack, I just quick-created this template based on the actual config I used ;-). Let me actually take those template strings from: https://github.com/atigyi/zephyr/blob/google_iot_device_sdk_integration/samples/net/google_iot_device/src/main.c#L23 (that comment otherwise needs to be deleted, as there's no longer command line).

Reply:
👍